### ISSUE TITLE
[Travis CI] Expected feature release number in range of 9 to 14, but got: 8

### ISSUE THREAD

**Issue by dubreuia** [dubreuia/intellij-plugin-save-actions#263]
```bash
$ export JAVA_HOME=~/oraclejdk8

$ export PATH="$JAVA_HOME/bin:$PATH"

$ ~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"

Ignoring license option: BCL -- using GPLv2+CE by default

install-jdk.sh 2019-09-17

Expected feature release number in range of 9 to 14, but got: 8

The command "~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"" failed and exited with 3 during .
```

**Comment by dubreuia** [comment]
JDK8 end of life

Workaround is to force trusty dist: https://travis-ci.community/t/error-installing-oraclejdk8-expected-feature-release-number-in-range-of-9-to-14-but-got-8/3766/4 (more info https://travis-ci.community/t/install-jdk-sh-failing-for-openjdk9-and-10/3998/3) but we'll have to update at some point.

**Comment by dubreuia** [comment]
Released in 1.6.0: https://plugins.jetbrains.com/plugin/7642-save-actions/versions
